pkg/azurelustreplugin/entrypoint.sh

```diff
@@ -41,6 +41,9 @@ function add_net_interfaces() {
     elif [[ "$interface_info" =~ 'link/ether' ]]; then
       echo "$(date -u) Including ethernet interface: ${interface}"
       ethernet_interfaces+=("$interface")
+    elif [[ "$interface_info" =~ 'UNKNOWN' ]]; then
```

Comment:
Is there a list of all (common) kinds of interfaces that we should consult and also exclude?   Token Ring, ATM, Sneakernet?